### PR TITLE
improve err msg for uploading volumes

### LINKED ISSUE #308: libvirt_volume: Error while uploading source os-image.qcow2: unexpected EOF
#  Version Reports:

### Distro  version of host:
Ubuntu
``` openSUSE 42.2/ Centos7/ Ubuntu.. ```
Ubuntu 18.04
### Terraform Version Report
(Provided by running `terraform -v`.)
0.11.7
### Libvirt version
libvirtd (libvirt) 4.0.0
```virsh --version```

### PR DESCRIPTION
**Pull request by MalloZup**
fix #308

### CODE REVIEW

**Comment by MalloZup** [comment]
@dmacvicar so at the end it ends up that was the image was not correct on the server were we download it. :smiley_cat:

**Comment by MalloZup** [comment]
@dmacvicar thank you for review, i will adapt the PR accordingly your comments. thx :tulip:

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ HIGHLIGHTS:
 
 * qemu-agent variable is not anymore env. variable, instead it is 
   domain terraform variable
-## 0.4.3 (August 14, 2018)
+
+# 0.4.3 (August 14, 2018)
```
Comment (flavio) [inline CHANGELOG.md]: I would make this fix with another dedicated PR, it's not relevant to this one. Plus it looks like other releases are still using a `h2` instead of a `h1`, it would be nice to be consistent within the whole file.

Comment (MalloZup) [reply CHANGELOG.md]: thx flavio i just overlooked the additional `#` :smile:  i was thinking i added only a new-line :smile: 

i will change this

**Comment by MalloZup** [comment]
@flavio thx for review. i left the changelog file untouched. merging
